src/stepper/index.tsx

```diff
@@ -97,6 +101,16 @@ export default class Stepper extends PureComponent <StepperProps, StepperState>
     this.update(this.state.value - this.props.step)
   }
 
+  createListeners(callback: () => void) {
+    return isTouchScreen ? {
+      onTouchStart: this.handleButtonDown(callback),
+      onTouchEnd: this.handleButtonUp(callback),
+    } : {
+      onMouseDown: this.handleButtonDown(callback),
+      onMouseUp:this.handleButtonUp(callback),
```

Comment:
Have you checked on hybrid devices, like a surface? Because `isTouchScreen` will return `true` so I'm thinking it might not even work if you use the mouse.

Reply:
Tested on a windows laptop with tactile screen, mouse + touch events works to increase/decrease value.